pilates/activitysim/preprocessor.py

```diff
@@ -134,7 +165,7 @@ def _transit_access(transit_df, access_paths, num_taz):
     ''' OD pair value for drive access '''
     df = transit_df.loc[transit_df.pathType.isin(access_paths), :].copy()
     df.drop_duplicates(['origin', 'destination'], keep='last', inplace=True)
-    assert df.shape[0] == num_taz * num_taz
+#     assert df.shape[0] == num_taz * num_taz
```

Comment:
this assert doesn't work because not all tazs have transit paths, right? I thought we were going to impute either 0 or really large travel times instead of leaving empty rows, though?